client/odysseus/message/index.tsx

```diff
@@ -1,5 +1,7 @@
 import classnames from 'classnames';
 import { RefObject, useState } from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Is there a better way of adding the package so we don't need to add this linter rule to skip?